### PR TITLE
Limit percentage counter's value to 2 decimal in perftab.

### PR DESCRIPTION
**Pull request by stellama0208**
Bug: N/A.

### CODE REVIEW

File: gapic/src/main/com/google/gapid/perfetto/Unit.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public Percent() {
 
       @Override
       public String format(double value) {
-        return (value == 100) ? super.format(100) : super.format(value);
+        return (value == 100) ? super.format(100) : String.format("%.2f%s", value, "%");
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/perfetto/Unit.java:204]: `String.format("%,.2f%%", value)`

File: gapic/src/main/com/google/gapid/perfetto/Unit.java [hunk 2]
```diff
@@ -195,13 +195,18 @@ public String format(double value) {
     }
 
     public static class Percent extends Simple {
+      private static final Double CUT_OFF = 0.05;
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/perfetto/Unit.java:198]: This should not be a boxed double. Also, add a blank line between the constant and the constructor.

File: gapic/src/main/com/google/gapid/perfetto/Unit.java [hunk 3]
```diff
@@ -195,13 +195,18 @@ public String format(double value) {
...
         super("%");
       }
 
       @Override
       public String format(double value) {
-        return (value == 100) ? super.format(100) : super.format(value);
+        if (value == 100) {
+          return super.format(100);
+        } else {
+          return Math.abs(value) < CUT_OFF ? String.format("%,.2g%%", value) : String.format("%,.2f%%", value);
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/perfetto/Unit.java:208]: You don't need to do the abs here, as it's done above - the formatter should only be getting positive values.
Also, the format should be `"%,g%%"`.

Comment (stellama0208) [reply gapic/src/main/com/google/gapid/perfetto/Unit.java]: Thanks for being patient here! Just done it! 👍

**Review by pmuetschard — APPROVED** [review]

**Pull request merged**